deploy/web_server/web_server.py

```
@@ -192,80 +205,195 @@ async def main(request):
+    if cache:
+        cache = cache.decode("utf-8")
+        if json_parse:
+            cache = json.loads(cache)
+
+    else:
+        cache = default_response
+
+    return cache
+
+
+async def get_alerts(mc, key_b, default_response={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)